src/resultsbottomuppage.cpp

```diff
@@ -82,7 +82,8 @@ ResultsBottomUpPage::ResultsBottomUpPage(FilterAndZoomStack* filterStack, PerfPa
     ui->setupUi(this);
 
     auto bottomUpCostModel = new BottomUpModel(this);
-    ResultsUtil::setupTreeView(ui->bottomUpTreeView, ui->bottomUpSearch, bottomUpCostModel);
+    ResultsUtil::setupTreeView<Data::BottomUp, BottomUpModel>(ui->bottomUpTreeView, ui->bottomUpSearch,
```

Comment:
by doing the change above, no change should be required here anymore and in the other similar hunks that follow